Assets/StreamingAssets/Saves.meta

```diff
@@ -0,0 +1,8 @@
+fileFormatVersion: 2
```

Comment:
Any reason for this folder its files to be where it is? We try to fit everything in one of the 4 classes (art, content, editor, engine), these seem fit for maybe the content folder instead of the asset folder directly?

Reply:
Unfortunately before the tilemap save file wasn't being included in builds. This folder is always being copied by Unity into a build. https://docs.unity3d.com/Manual/StreamingAssets.html

Reply:
So the StreamingAssets folder is required to be in this location and cannot be moved?

Reply:
Yeah it's generated by Unity automatically. Though I'm not sure if another method exists to include the tilemap save file with builds.